pkg/skaffold/initializer/init.go

```diff
@@ -221,7 +222,19 @@ func autoSelectBuilders(builderConfigs []InitBuilder, images []string) ([]builde
 	return pairs, builderConfigs, unresolvedImages
 }
 
-func detectBuilders(path string) ([]InitBuilder, error) {
+func detectBuilders(enableJibInit bool, path string) ([]InitBuilder, error) {
+	// TODO: Remove backwards compatibility if statement (not entire block)
+	if enableJibInit {
+		// Check for jib
+		if builders := jib.ValidateJibConfigFunc(path); builders != nil {
```

Comment:
Isn't `ValidateJibConfig` more natural, given that `...Func` is like an ad-hoc variable only used by testing?

Reply:
I'm having trouble finding an example now, but I think I noticed this was a convention used elsewhere in the codebase.